### PR TITLE
feat(date component): Seperate date component to reuse

### PR DESCRIPTION
**Pull request by YoonYeoHwan**

### CODE REVIEW

File: web/src/components/dateTime.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    super(props);
+    this.state = {
+      schedule: this.props.children,
+    };
+  }
+  render() {
+    const date = new Date(this.state.schedule.startDateTime);
+    const month = date.toLocaleString('US', {month: 'long'}).substring(0, 3);
+    const year = date.toLocaleString('US', {year: 'numeric'});
+    const day = date.toLocaleString('US', {day: '2-digit'});
```
Comment (shberry) [inline web/src/components/dateTime.js]: Does this mean 1~9 will be 01, 02, ... ,09 ?

File: web/src/components/dateTime.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    const options = {
+      hour: '2-digit',
```
Comment (shberry) [inline web/src/components/dateTime.js]: Same here - if it's 1:10, this will be displayed as 01:01?

File: web/src/components/dateTime.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      minute: '2-digit',
+    };
+    const time = date.toLocaleString('US', options);
```
Comment (shberry) [inline web/src/components/dateTime.js]: Maybe add a comment with an example date? So people can understand the format without reading the code. How about jsing January 1st, 2020 1:01 am?

File: web/src/components/dateTime.js (hunk 1 above)
Comment (YoonYeoHwan) [reply web/src/components/dateTime.js]: Yes. It's right

**Pull request merged**